cedar-lean/Cedar/Data/List.lean

```diff
@@ -51,19 +51,6 @@ def canonicalize [LT β] [DecidableLT β] (f : α → β) : List α → List α
   | [] => []
   | hd :: tl => insertCanonical f hd (canonicalize f tl)
 
-def Equiv {α} (a b : List α) : Prop :=
-  a ⊆ b ∧ b ⊆ a
-
-infix:50 " ≡ " => Equiv
```

Comment:
don't we still need this definition in Cedar.Data?  Or is it really only used in proofs?

Reply:
It turns out it's used only in proofs.  I was surprised too and was on the fence about leaving it in `Cedar.Data.List` but decided to stick with putting all proof-only things into `Cedar.Thm.*`.  

If we end up wanting to use it in code too, we can move it back (and probably also need to prove that `Equiv a b` is Decidable).